src/Api/Startup.cs

```diff
@@ -206,7 +212,20 @@ public void ConfigureServices(IServiceCollection services)
         app.UseMiddleware<CurrentContextMiddleware>();
 
         // Add endpoints to the request pipeline.
-        app.UseEndpoints(endpoints => endpoints.MapDefaultControllerRoute());
+        app.UseEndpoints(endpoints =>
+        {
+            endpoints.MapDefaultControllerRoute();
+
+            endpoints.MapHealthChecks("/health/simple");
```

Comment:
We should follow the psuedo `/healthz` standard. It's the example in .NET and k8s documentation so might as well.